mantaray/views.py

```diff
@@ -6,6 +6,7 @@
 import sys
 import tkinter
 import subprocess
+from playsound import playsound  # type: ignore
```

Comment:
Would you like to contribute stubs for this in typeshed? I can guide you through how that works, and also check whether our documentation about this is up to date. There are tools to do most of the work for you.

Reply:
Sure! Should be simple. `playsound` only has one single function :)

Reply:
Our documentation seems to be terribly outdated. We have a new script that makes this kind of thing very easy, but it's broken with the latest mypy not documented. I made a pull request that fixes it. Once you have made the playsound stubs, would you like to make a pull request that also documents the process? :D

Here's how the process should work:
- Fork and clone typeshed. Make a new branch.
- Make a venv and `pip install -r requirements-tests.txt`
- Install playsound with pip
- Run `python3 scripts/create_baseline_stubs.py playsound`. Follow the instructions.